OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -71,23 +71,19 @@ public class ProductionQueue : IResolveOrder, ITick, ITechTreeElement, INotifyOw
 
 		// A list of things we could possibly build
 		readonly Dictionary<ActorInfo, ProductionState> producible = new Dictionary<ActorInfo, ProductionState>();
-		readonly List<ProductionItem> queue = new List<ProductionItem>();
+		protected readonly List<ProductionItem> Queue = new List<ProductionItem>();
```

Comment:
Why make it start with a capital letter? It isnt public.

Reply:
Im sure "make check" told me it has to be uppercase. I'll verify that and change it if not.

Reply:
OpenRA.Mods.Common\Traits\Player\ProductionQueue.cs:L74: [SA1304] Readonly variables that are not declared private must start with an upper-case letter: queue.